### ISSUE TITLE
In the new html tooltips, is there a way to show a fas icon?

### ISSUE THREAD

**Issue by nrshapiro** [Psychoanalytic-Electronic-Publishing/PEP-Web-User-Interface#503]
The new html tooltips work great.  

I've worked through most of them, but on Related Documents, there's the text {falcon} embedded in the help text for the Related Articles Infocard.  I don't know if that ever showed the right icon...is there a way to show it now that we have some HTML abilities?

I tried the `<i class="fas fa-layer-group"></i>
`
not sure that's the right icon even., but it doesn't work anyway, I presume because I can't add the header include.

I searched for font-awesome ember usage, and saw this as a possible format which was similar to what was there, but it doesn't work here either.

`{{fa-icon "coffee"}}`

**Comment by nrshapiro** [comment]
Also, not sure if it will cause other problems, but can we use a black backgound with white text like before?

**Comment by bakerac4** [comment]
@nrshapiro I can make it a black background if you would like. You _technicallly_ can use FA icons in those tooltips but its convoluted. The newer versions of font awesome use SVG icons by default. Thats what we use, so when its generated it looks something like 

```<svg viewBox="0 0 576 512" xmlns="http://www.w3.org/2000/svg" role="img" focusable="false" aria-hidden="true" data-icon="home" data-prefix="fal" id="ember8" class="svg-inline--fa fa-home fa-w-18 fa-lg ember-view"><path fill="currentColor" d="M541 229.16l-61-49.83v-77.4a6 6 0 0 0-6-6h-20a6 6 0 0 0-6 6v51.33L308.19 39.14a32.16 32.16 0 0 0-40.38 0L35 229.16a8 8 0 0 0-1.16 11.24l10.1 12.41a8 8 0 0 0 11.2 1.19L96 220.62v243a16 16 0 0 0 16 16h128a16 16 0 0 0 16-16v-128l64 .3V464a16 16 0 0 0 16 16l128-.33a16 16 0 0 0 16-16V220.62L520.86 254a8 8 0 0 0 11.25-1.16l10.1-12.41a8 8 0 0 0-1.21-11.27zm-93.11 218.59h.1l-96 .3V319.88a16.05 16.05 0 0 0-15.95-16l-96-.27a16 16 0 0 0-16.05 16v128.14H128V194.51L288 63.94l160 130.57z"></path>
</svg>```

**Comment by bakerac4** [comment]
@nrshapiro just confirming before I start, you want me to change the background color of the tooltips?

**Comment by nrshapiro** [comment]
Yes, if it's a quick one or two line switch so it can be reverted if it doesn't look good with the new longer text.  If it's more than 10 minutes work, no.

Is it at all sensitive to the other colors?  What happens in the Dark theme?

**Comment by nrshapiro** [comment]
@bakerac4 Thanks that works, and that's good enough for now, while I am doing some of the editing.  but I suspect the future "moderator" is not going to be happy that the code for that help topic now looks like:

For documents which have related articles, the <svg aria-hidden="true" focusable="false" data-prefix="fas" data-icon="layer-group" class="svg-inline--fa fa-layer-group fa-w-16" role="img" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 512 512"><path fill="currentColor" d="M12.41 148.02l232.94 105.67c6.8 3.09 14.49 3.09 21.29 0l232.94-105.67c16.55-7.51 16.55-32.52 0-40.03L266.65 2.31a25.607 25.607 0 0 0-21.29 0L12.41 107.98c-16.55 7.51-16.55 32.53 0 40.04zm487.18 88.28l-58.09-26.33-161.64 73.27c-7.56 3.43-15.59 5.17-23.86 5.17s-16.29-1.74-23.86-5.17L70.51 209.97l-58.1 26.33c-16.55 7.5-16.55 32.5 0 40l232.94 105.59c6.8 3.08 14.49 3.08 21.29 0L499.59 276.3c16.55-7.5 16.55-32.5 0-40zm0 127.8l-57.87-26.23-161.86 73.37c-7.56 3.43-15.59 5.17-23.86 5.17s-16.29-1.74-23.86-5.17L70.29 337.87 12.41 364.1c-16.55 7.5-16.55 32.5 0 40l232.94 105.59c6.8 3.08 14.49 3.08 21.29 0L499.59 404.1c16.55-7.5 16.55-32.5 0-40z"></path></svg>  icon will show up next to listing, indicating that the article is part of a series or has follow up commentary in the journal.  

instead of something more simple like:

For documents which have related articles, the <i class="fas fa-layer-group"></i>  icon will show up next to listing, indicating that the article is part of a series or has follow up commentary in the journal.  

Is that not possible, as it is in ordinary HTML with the proper Font Awesome style sheets loaded in the header?

(Please answer the question, but I will put the issue in Limbo if more work is needed after that.)
